pkg/apis/core/v1/resource_test.go

```diff
@@ -48,6 +49,91 @@ var _ = Describe("resource.Resource", func() {
 		)
 	})
 
+	Context("RetryResourceTagging", func() {
+		var a api.API
+		var srv *ghttp.Server
+		BeforeEach(func() {
+			srv = ghttp.NewServer()
```

Comment:
We should probably close the server afterwards. See https://onsi.github.io/gomega/#making-assertions-against-an-incoming-request